desktop/onionshare/connection_tab.py

```diff
@@ -326,11 +326,20 @@ def tor_con_fail(self, msg):
         """
         self.tor_con.hide()
 
-        # If we're on first launch, switch to use bridge
+        # If there is a message, update the text of the bridge widget
+        if msg:
+            self.use_bridge_widget.connection_error_message.setText(f"{msg}")
```

Comment:
Is `(f"{msg}")` typical of best practice (given there's no mix of strings being interpolated, I would normally just do `(msg)` but maybe this is not right)

Reply:
I was actually doing some string interpolation, which I then removed, but forgot to remove the f-string format. My mistake.